recipe/0002-Fix-incompatible-function-pointer-types-add-CONST86.patch

```diff
@@ -0,0 +1,128 @@
+From 517fa24afda62237bdb4d3de126769f40c853bfe Mon Sep 17 00:00:00 2001
+From: Marcel Bargull <marcel.bargull@udo.edu>
+Date: Sat, 22 Jun 2024 14:40:27 +0000
+Subject: [PATCH] Fix incompatible-function-pointer-types, add CONST86
+
+---
+ generic/tkTable.h     | 7 +++++--
+ generic/tkTableTag.c  | 4 ++--
+ generic/tkTableUtil.c | 8 ++++----
+ generic/tkTableWin.c  | 4 ++--
+ 4 files changed, 13 insertions(+), 10 deletions(-)
+
+diff --git a/generic/tkTable.h b/generic/tkTable.h
+index cfb83e7..827debb 100644
+--- a/generic/tkTable.h
++++ b/generic/tkTable.h
+@@ -40,6 +40,9 @@
+ #ifndef CONST84
+ #  define CONST84
+ #endif
++#ifndef CONST86
++# define CONST86 CONST84
++#endif
```

Comment:
`Tk_CustomOption`'s member `Tk_OptionPrintProc` from Tk 8.6's `tk.h` has its return type defined as `CONST86 char *`.
This patch adds that `const`ness to the functions that are used as `Tk_OptionPrintProc`.